packages/core/event.js

```diff
@@ -6,10 +6,10 @@ const jsRuntime = require('./lib/js-runtime')
 const metadataDelegate = require('./lib/metadata-delegate')
 const isError = require('./lib/iserror')
 
-class BugsnagEvent {
+class Event {
   constructor (errorClass, errorMessage, stacktrace = [], handledState = defaultHandledState(), originalError) {
     // duck-typing ftw >_<
-    this.__isBugsnagEvent = true
+    this.__isEvent = true
```

Comment:
Where is this used?

Reply:
Good shout! It's no longer used since #664.